### ISSUE TITLE
Make streaming API / websocket information more visible

### ISSUE THREAD

**Issue by andypiper** [mastodon/documentation#1237]
Per [comments in this issue](https://github.com/mastodon/mastodon/pull/25532#discussion_r1242383615), the [websocket API](https://docs.joinmastodon.org/methods/streaming/#websocket) is currently only really findable via Google search - needs to be surfaced more obviously.

**Comment by ThisIsMissEm** [comment]
It's currently under API Methods -> Timelines -> Streaming, but it's not actually part of the REST API, so lifting it up to up near REST APIs (perhaps just "APIs") might make more sense.

**Comment by mjankowski** [comment]
> i don't think we distinguish between REST and non-REST (whatever that would look like)

For streaming specifically, there's one endpoint which either returns 404 not found or redirects to the streaming url when on a different host. I'd argue this is in "the REST API" and not the streaming API ... though I generally agree that drawing these lines might not be useful. Was mostly trying to sort out how to resolve this issue.

> the confusion seems to have been about nesting the `streaming` methods indented underneath `timelines`, which was done because streaming is conceptually related to timelines.

Would making this change (move it out of timelines) resolve this issue?

> i think the better solution is to make the theme support search, and make the sidebar collapsible so that it's not so intimidating.

I'm not sure how to get an opinion from anyone on the desirability of things like search, theme changes, collapse, etc - but I agree those would all be improvements.

> tangentially, it seems that some new methods were added that could be conceptually grouped together [...]

They could ... there's sort of a mix of grouping just by API method endpoint/path, or grouping thematically.

**Comment by mjankowski** [comment]
Thanks for that summary and clarification of where things are at. Those are all good goals/improvements!

At risk of further derailing -- and maybe this is way too nuanced a question -- but just to be clear:

- Are we in a "hard no" scenario, where nothing along these lines will be at all contemplated/reviewed/etc (until the team can eventually do this in a coordinated way, "correctly" in-house), and where some issues/PRs can probably close? https://github.com/mastodon/documentation/issues/1068 / https://github.com/mastodon/documentation/issues/1075 / https://github.com/mastodon/documentation/issues/1443 / https://github.com/mastodon/documentation/pull/1061 / https://github.com/mastodon/documentation/pull/1130 / maybe more?
- More of a "soft no", where efforts at light/dark mode, search, sidebar, etc - MIGHT be considered ... but only if easy to review and only if they kept things __on the path__ to those larger goals?

Basically ... should we close all current "docs site meta issues" sort of stuff and essentially only entertain content changes for now? (maybe update README and/or contribute guide along those lines...)
